MapCore/src/main/java/org/wwarn/mapcore/client/components/customwidgets/facet/FacetSearchableCheckBoxWidget.ui.xml

```diff
@@ -38,25 +38,46 @@
 
         .checkBoxes {
             border: 1px solid #ccc;
-            width: 240px!important;
+            width: 240px !important;
         }
 
-        .facetCheckBox a, .facetCheckBox a.clearLink, a.clearLink {
-            float: right!important;
-            font-size: 1em!important;
-            font-weight: bold!important;
+        .facetCheckBox a.searchableCheckBoxClearAnchor, .facetCheckBox a.clearLink, a.clearLink {
+            float: right !important;
+            font-size: 1em !important;
+            font-weight: bold !important;
         }
 
-        .facetCheckBoxPanel{
-            display: none!important;
+        .facetCheckBoxPanel {
+            display: none !important;
         }
 
         .facetCheckBox input[readonly], .facetCheckBox .disabledCheckBox,
-        .searchCheckBoxListItemDisabled label{
-            color: #aaa!important;
+        .searchCheckBoxListItemDisabled label {
+            color: #aaa !important;
+        }
+
+        .facetCheckBox {
+            /*display: ;*/
+        }
+
+        .subHeading {
+            width: 100%;
         }
-    </ui:style>
-    <g:VerticalPanel addStyleNames="{style.facetCheckBox}">
 
+        .toggleFilterHref {
+            float: left !important;
+
+        }
+
+        .searchableCheckBoxClearAnchor {
+
```

Comment:
Okay, I have removed this, as it isn't needed technically anymore, since I have moved the markup around.